content/article/analysing-fit-data-with-perl-producing-png-plots.md

```diff
@@ -0,0 +1,989 @@
+
+  {
+    "title"       : "Analysing FIT data with Perl: producing PNG plots",
+    "authors"     : ["paul-cochrane"],
+    "date"        : "2025-06-19T12:14:08",
+    "tags"        : ["science", "sport", "cycling"],
+    "draft"       : false,
+    "image"       : "/images/analysing-fit-data-with-perl/plot-with-perl-logos-and-cyclist-cover.png",
+    "thumbnail"   : "/images/analysing-fit-data-with-perl/plot-with-perl-logos-and-cyclist.png",
+    "description" : "",
```

Comment:
Could we get a description for this?

Reply:
Sure! Weird that I forgot this as you'd only recently asked me about that for another article and I thought I was taking care.  Oops!